10_Examples/linh-onboarding/include/linh-onboarding/subscriber.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <memory>
+#include "std_msgs/msg/string.hpp"
+#include "std_msgs/msg/float32.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+using std::placeholders::_1;
```

Comment:
"using" statements are cool so that you can use shorter names for variables and methods and stuff, but we want to avoid putting them in header files.

The reason is, if other people include your header file, then the using statement also gets included. If they have some variable named _1 for some reason, it will get very confusing very fast.

I imagine this probably got yoinked over from the example code and so its probably my bad that its in a header xD.